src/cli/options.ts

```diff
@@ -46,6 +46,16 @@ export function withStateOption<T>(
   });
 }
 
+export function withIdOption<T>(
+  yargs: Argv<T>
+): Argv<T & { id?: string }> {
+  return option(yargs, "id", {
+    string: true,
+    describe: "select snippets with a specific id",
+    once: true,
```

Comment:
Is there a reason not to support id multiple times? (To allow a set of IDs to be snipped)

Reply:
Great point. I agree this should support multiple ids. I will try and implement this and get this back to you.